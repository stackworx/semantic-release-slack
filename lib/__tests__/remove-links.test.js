const removeLinks = require('../remove-links');

const notes =
  '## [1.0.5](https://gitlab.com/stackworx.io/merchant-portal/compare/v1.0.4...v1.0.5) (2018-08-08)\n\n\n### Bug Fixes\n\n* Resolves BLMP-143 Testing release ([d1a3a89](https://gitlab.com/stackworx.io/merchant-portal/commit/d1a3a89))\n\n\n\n';

test('removeLinks', () => {
  expect(removeLinks(notes)).toMatchSnapshot();
});
