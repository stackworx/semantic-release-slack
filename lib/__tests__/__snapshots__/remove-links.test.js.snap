// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`removeLinks 1`] = `
"## 1.0.5 (2018-08-08)


### Bug Fixes

* Resolves BLMP-143 Testing release (d1a3a89)



"
`;
